pyproject.toml

```diff
@@ -17,6 +17,7 @@ readme = "README.md"
 dynamic = ["version"]
 
 dependencies = [
+    "pydantic == 2.6.0"
```

Comment:
If we up pydantic to 2.6, python should be bumped to >=3.8

Reply:
updated